references/endpoints/Create_Room_Pages.mdx

```diff
@@ -0,0 +1,30 @@
+---
+openapi: "POST /rooms/{room_id}/pages/create_many"
+---
+
+Currently, You cannot try this API on the docs.
```

Comment:
Mintlify seems not to be able to process a requestBody of an array type.
Added manual guide to replace it.